versioned_docs/version-0.1.0/advanced-topics/messaging-system.md

```diff
@@ -1,8 +1,13 @@
 ---
 id: messaging-system
 title: Messaging System and RPCs
+description: An introduction to PRCs and custom messages in MLAPI
 ---
 
+The MLAPI has two parts to it's messaging system. Remote Procedure Call (RPC) messages and Custom Messages. Both types have sub types that change their behaviour, functionality and performance.
```

Comment:
/docs version has links, so mirroring that new content for here. Make sure to add custom-messaging file to /docs (develop)
```suggestion
The MLAPI has two parts to its messaging system: [RPC messages](about-rpc.md) and [Custom Messages](custom-messages.md). Both types have sub-types that change their behaviour, functionality, and performance.
```